login-workflow/src/screens/SelfRegistrationPager.tsx

```diff
@@ -219,6 +219,7 @@ export const SelfRegistrationPager: React.FC = () => {
                 <AcceptEula
                     eulaAccepted={eulaAccepted}
                     onEulaChanged={setEulaAccepted}
+                    // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
Why is this a misused promise?

Reply:
**loadEula** accepts void return and we are returning a Promise. That's why I disabled eslint here. If we change the return type of loadEula, we are not using it as a promise.

Reply:
Do you think it makes more sense to put the eslint disable here or to update the type, i.e.:
![image](https://user-images.githubusercontent.com/29152776/161058392-7f740165-4b7a-4777-98dd-34d9cdaa7434.png)

Reply:
Yes, I tried updating the loadEula return type to ()=>Promise<void> | void; but forgot to use void in call. Changes have been updated.